Gemfile

```diff
@@ -63,6 +63,21 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of chromedriver to run system tests with Chrome
   gem 'chromedriver-helper'
+
+  # Clean database on test
+  gem 'database_cleaner'
+
+  # Fake data to test
+  gem 'faker'
```

Comment:
you maaaay want to change the faker and the factory_bots_rails to the `group :development, :test`, believe me, in development you will be able to create records  in the Rails Console that could be very helpful to tests thing on your own